README.md

```diff
@@ -168,23 +196,36 @@ It is also possible to create models for selected checkpoints only. Examples:
 
 Add `MODEL_NAME` and `OUTPUT_DIR` and replace `data/foo` with the output directory if needed.
 
-## Plotting CER (experimental)
+### Plotting CER
 
-Training and Evaluation CER can be plotted using Matplotlib. A couple of scripts are provided
-as a starting point in the `plot` subdirectory for plotting different training scenarios. The training
-log is expected to be saved in `plot/TESSTRAIN.LOG`.
+Training and Evaluation Character Error Rate (CER) can be plotted using Matplotlib:
 
-As an example, use the training data provided in 
-[ocrd-testset.zip](./ocrd-testset.zip) to do training and generate the plots.
-Plotting can be done while training is running also to depict the training status till then.
-```
-unzip ocrd-testset.zip -d data/ocrd-ground-truth
-nohup make training MODEL_NAME=ocrd START_MODEL=deu_latf TESSDATA=~/tessdata_best MAX_ITERATIONS=10000 > plot/TESSTRAIN.LOG &
-```
-```
-cd ./plot
-./plot_cer.sh 
-```
+    # Make OUTPUT_DIR/MODEL_FILE.plot_*.png
+    make plot
+
+All the variables defined above apply, but there is no explicit dependency on `training`.
+
+Still, the target hinges on the `LOG_FILE` intercepted during training (just will not trigger
+training itself). Besides the log file, this also evaluates the trained models (for each checkpoint)
+on the eval dataset. The latter is also available as an independent target `evaluation`:
```

Comment:
```suggestion
Still, the target will depends on the `LOG_FILE` captured during training (it will just not trigger training itself).
Besides the log file, the trained models (for each checkpoint) are also evaluated on the eval dataset.
The latter is also available as an independent target `evaluation`:
```

Reply:
`will depends`?

I don't like passive voice here, but pushed an improved paragraph.